header-only/include/awkward/GrowableBuffer.h

```diff
@@ -408,6 +414,7 @@ namespace awkward {
       panel_ = std::move(std::unique_ptr<Panel<PRIMITIVE>>(
           new Panel<PRIMITIVE>((size_t)options_.initial())));
       ptr_ = panel_.get();
+      length_ = 0;
     }
```

Comment:
This looks like a bug-fix. Is it?

Reply:
> I've always been a little confused about what std::move does. It seems pretty clear (in this example and others) that it copies data.

I can chime in here! `std::move` just indicates to the compiler that the argument can be moved from, i.e. a consumer of the `std::move(...)` could avoid a copy by moving from the argument. So, it doesn't always eliminate a copy, depending upon how it is consumed. It's just a signal of lifetime intent IIRC.

Reply:
> This looks like a bug-fix. Is it?

Yes, it is.